src/pages/Listeners.vue

```diff
@@ -44,24 +96,42 @@ export default {
           // sortable: true,
         },
       ],
+      selectedIds: [],
+      pagination: { rowsPerPage: 20 },
+      isConfirmDeleteModalOpen: false,
+      isCreateModalOpen: false,
+      editingMode: false,
+      id: '',
+      email: '',
+      name: '',
+      surname: '',
     };
   },
   computed: {
     ...mapState({
       listenersList: state => state.listeners.listeners,
+      listener: state => state.listeners.listener,
       loading: state => state.listeners.loading,
       seminarsList: state => state.seminars.seminars.map(seminar => ({
```

Comment:
have to use [mapGetters](https://vuex.vuejs.org/api/#mapgetters)

Reply:
Done.